src/modules/services_manager_service.rb

```diff
@@ -182,7 +180,7 @@ def extract_services
           log.info "  have #{sh}; found #{s.inspect}"
           sh[:enabled] = s && s.enabled?
           log.info "  enabled #{sh[:enabled]}"
-          sh[:active] = is_active?(name)
+          sh[:active] = s && s.active?
```

Comment:
just be sure it does not break in installation, as if I remember correctly we introduce that is-active for some broken (not yet known ) states.

Reply:
Yes, the old fix is preserved: https://github.com/yast/yast-yast2/pull/613/files#diff-f44cd7355b2160b663b0bb24f66dbe9aR40